decorater.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'nameable'
+require_relative 'person'
+
+class Decorator
```

Comment:
[REQUIRED] Kindly ensure that the Decorator class inherits from Nameable. This is a requirement.

Reply:
Okay, I will add inheritance